src/smokeTest/java/uk/gov/hmcts/futurehearings/hmi/smoke/updateuser/UpdateUserApiSmokeTest.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.futurehearings.hmi.smoke.updateuser;
+
+import static io.restassured.RestAssured.expect;
+
+import uk.gov.hmcts.futurehearings.hmi.Application;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+
+@SpringBootTest(classes = {Application.class})
+@ActiveProfiles("smoke")
+public class UpdateUserApiSmokeTest {
```

Comment:
refactor to User rather than updateUser as we should name based on the resource rather than operation nd Resource wish is a SOAP Style of coding....